### PR TITLE
Added Favicon

### PR DESCRIPTION
**Pull request by avichal04**
Favicons are now added to `welcome.html` , `404.html`, `restaurant.html`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,6 +19,7 @@ __pycache__/
 # C extensions
 *.so
 
+
```
Comment (marcpage) [inline .gitignore:22]: If the only change in this file is an added blank line, let's remove the blank line.

**Review by marcpage — DISMISSED** [review]
`e.ico`, `error404.png`, and `r.ico` should be in `src/ui`. Please `git mv` them to `src/ui` and push those changes.

File: .gitignore (hunk 1 above)
Comment (avichal04) [reply .gitignore:22]: I have changed nothing in this file.
Kindly remove the blank if possible .

Comment (marcpage) [reply .gitignore:22]: You can remove the blank line in your source and then commit and push the changes.

Comment (avichal04) [reply .gitignore:22]: Sir can you do me a favour?

Comment (avichal04) [reply .gitignore:22]: I am having some issues with my git!
I want help regarding that.

**Comment by avichal04** [comment]
> `e.ico`, `error404.png`, and `r.ico` should be in `src/ui`. Please `git mv` them to `src/ui` and push those changes.

Did it.

**Review by marcpage — APPROVED** [review]
Looks great. Thank you.

**Pull request merged**